Drivers/inc/LTC2630.hpp

```diff
@@ -13,30 +13,33 @@
 namespace SolarGators {
 namespace Drivers {
 
+
+typedef enum class OperatingMode : uint8_t{
+  Bit8 = 0,
+  Bit10 = 1,
+  Bit12 = 2
+} OperatingMode;
+
 class LTC2630 {
   SPI_HandleTypeDef* hspi_;
+  GPIO_TypeDef* cs_port_;
+  uint16_t cs_pin_;
   uint16_t current_val_;
   uint16_t pending_val_;
-  uint8_t mode_;
-  namespace Command
```

Comment:
Just curious why you dropped the `namespace` here

Reply:
It made it look nicer when not using the namespace, that should've never been on main I was just too lazy to pick the commit off.